examples/blink/CMakeLists.txt

```diff
@@ -2,6 +2,17 @@ cmake_minimum_required(VERSION 3.12)
 
 set(ROOT_DIR ${CMAKE_CURRENT_LIST_DIR}/../../)
 set(CMAKE_TOOLCHAIN_FILE ${ROOT_DIR}/toolchain-rv32.cmake)
+# ESP Target MCU. Supported: esp32c3, esp32h2
+list(APPEND supported_mcu esp32c3 esp32h2)
+
+if(target IN_LIST supported_mcu)
```

Comment:
It might make sense to handle the case of `target` not being set, e.g. by erroring out or defaulting to one of the supported targets

Reply:
The main idea here was to get you opinion about argument `-D target=esp32c3` or `-D target=esp32h2` to configure the exmaple for specific target. 

What do you think about that idea? Is there any better way to do that?

Meanwhile, I'll add handling of `target` not being set.

Reply:
The approach with adding a variable to set the chip target looks fine to me!